docker/test-images/zipkin-elasticsearch8/Dockerfile

```diff
@@ -26,13 +26,13 @@ FROM scratch as scratch
 
```

Comment:
@codefromthecrypt could we try with:

```
ARG java_version= 21.0.1_p12
```

There is a regression in 21.0.2 (https://github.com/elastic/elasticsearch/pull/104347) which should be fixed in 8.12.1

Reply:
cool trying now!

Reply:
totally works. not sure how you figured this out, but so glad you are on the team!

Reply:
ES 7 already sorted it, so bumping the version https://github.com/elastic/elasticsearch/commits/v7.17.17

Reply:
> totally works. not sure how you figured this out, but so glad you are on the team!

I just knew that from OpenSearch (since we largely have a same codebase and I was working in mitigation this week)

Reply:
guess I hadn't properly doxxed you ;) I had no idea me randomly asking you to help with elasticsearch was not so random.